appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/PayaraMicroImpl.java

```diff
@@ -1043,19 +1043,14 @@ public PayaraMicroRuntime bootStrap() throws BootstrapException {
 
             // boot the server
             preBootCommands.executeCommands(gf.getCommandRunner());
-            HazelcastCore.setThreadLocalDisabled(true);
-            try {
-                gf.start();
+            gf.start();
 
-                // Execute post boot commands
-                postBootCommands.executeCommands(gf.getCommandRunner());
-                this.runtime = new PayaraMicroRuntimeImpl(gf, gfruntime);
+            // Execute post boot commands
+            postBootCommands.executeCommands(gf.getCommandRunner());
+            this.runtime = new PayaraMicroRuntimeImpl(gf, gfruntime);
 
-                // load all applications, but do not start them until Hazelcast gets a chance to initialize
-                deployAll();
-            } finally {
-                HazelcastCore.setThreadLocalDisabled(false);
-            }
+            // load all applications, but do not start them until Hazelcast gets a chance to initialize
+            deployAll();
             if (!noCluster) {
                 gf.getCommandRunner().run("set-hazelcast-configuration", "--enabled", "true", "--dynamic", "true", "--target", "server-config", "--hostawarepartitioning", Boolean.toString(hostAware), "--lite", Boolean.toString(liteMember));
             }
```

Comment:
If all the reproducers we mentioned work with this configuration, would you let try swapping these two statements, finalizing config before deploy?

Reply:
I've swapped them and my reproducer seems to work fine, so I agree this is a good change. I'm going to commit it, I just need to test the other reproducers before I merge this